Eatery/Controllers/BRB/AboutTableViewController.swift

```diff
@@ -55,6 +56,15 @@ class AboutTableViewController: UITableViewController {
         tableView.register(LogoutTableViewCell.self, forCellReuseIdentifier: CellIdentifier.logout.rawValue)
     }
 
+    // Detects when user taps the 'Back' button in navigation bar
+    // Source: https://stackoverflow.com/a/32245222
+    override func willMove(toParentViewController parent: UIViewController?) {
+        super.willMove(toParentViewController: parent)
+        if parent == nil {
```

Comment:
Newline after `super. ...`.

But I think you might be able to cut this because the delegate is always `nil` (*verify that*).